src/modules/hmisAppSettings/Provider.tsx

```diff
@@ -56,7 +56,7 @@ export const HmisAppSettingsProvider: React.FC<Props> = ({ children }) => {
     setLoading(true);
     return logout()
       .then(() => {
-        reloadWindow();
+        window.location.assign('/');
```

Comment:
Navigate to root to avoid saving the current url for redirect after login, assuming that's not desired.

Reply:
yeah good call, thanks for fixing that